<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_input">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>导入文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lex">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>词法分析</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gram">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>语法分析</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_format">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>中间代码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_targin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>目标代码</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>1240</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>622</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>信息：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_source">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>731</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_table">
   <property name="geometry">
    <rect>
     <x>769</x>
     <y>69</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_format">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>340</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_lex">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_gram">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>340</y>
     <width>461</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_targin">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>340</y>
     <width>221</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lable_source">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>源程序</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable_table">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>符号表</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable_lex">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>词法分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gram">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>语法分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_format">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>310</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中间代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_targin">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>310</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目标代码</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
